Lib/_pyrepl/console.py

```diff
@@ -173,8 +180,11 @@ def _excepthook(self, typ, value, tb):
         self.write(''.join(lines))
 
     def runsource(self, source, filename="<input>", symbol="single"):
+        if self.barry_as_FLUFL:
+            source = source.replace("<>", "!=")
```

Comment:
Too simplistic, `!=` must become a syntax error:

```
>>> 1 != 2
  File "<stdin>", line 1
    1 != 2
      ^^
SyntaxError: with Barry as BDFL, use '<>' instead of '!='
```